Add tranform to normalize water `tunnel`s

rmarianski opened the pull request:
Refs https://github.com/mapzen/vector-datasource/issues/156

rmarianski committed:
Add tranform to normalize water `tunnel`s

rmarianski commented:
@zerebubuth Could you review please?

zerebubuth commented on TileStache/Goodies/VecTiles/transform.py:
```diff
@@ -337,6 +337,16 @@ def place_ne_capital(shape, properties, fid, zoom):
 
+def water_tunnel(shape, properties, fid, zoom):
+    tunnel = properties.get('tunnel')
+    if tunnel in ('yes', 'true'):
```
There seems to be a wide diversity of values [in taginfo for tunnel](http://taginfo.openstreetmap.org/keys/tunnel#values), almost a third are tagged `tunnel=culvert` and a smaller number are `tunnel=building_passage`. While we probably don't (yet) care about all these different types of tunnel, it seems there's a lot more values indicating some sort of tunnel than indicating no tunnel. Might it be better to reverse the logic and say `if tunnel not in ('no', 'false', '0', None)`?

rmarianski replied:
Great point. I've updated the logic.

rmarianski committed:
Reverse tunnel value logic

There are more valid values for tunnels, and it's better to check if
something is not a tunnel instead.

zerebubuth commented:
:+1: looks good to me!
